dockerfiles/haproxy-static-ingress/haproxy.cfg.tpl

```diff
@@ -12,11 +12,25 @@ defaults
     option tcplog
     maxconn 10000
 
+resolvers vpcdns
+    nameserver vpc ${RESOLVER}
+    resolve_retries 3
+    timeout resolve 1s
+    timeout retry   1s
+    hold valid 1s
+    hold timeout 1s
+    hold nx 1s
+    hold other 1s
+    hold refused 1s
+    hold obsolete 1s
+
 frontend nlb
-    bind *:4500
+    mode tcp
+    bind *:$PORT
     default_backend alb
 
 backend alb
+    mode tcp
     balance roundrobin
     default-server check maxconn 200
-    $BACKENDS
+    server alb $BACKEND:$PORT resolvers vpcdns check inter 100 fastinter 100
```

Comment:
The `inter` parameter is the interval between 2 consecutive health checks to 2000 ms (default). This line sets it to 100 ms.  Is it okay to have 0.1s?

Reply:
I think it should be, it is just doing a TCP healthcheck which runs internally which should be very fast